src/Order.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import RestaurantButton from "./RestaurantButton";
+
+const Order = props => {
+  const [pizzas, setPizzas] = useState(0);
+  const [orders, setOrders] = useState(0);
+  const [price, setPrice] = useState(0);
+  const [bill, setBill] = useState(price);
+
+  const orderOne = () => {
+    setOrders(orders + 1);
+    setPrice(props.price + price);
```

Comment:
Nice work, I like the addition of `price` state 🙂

Reply:
thanks. I was trying to make Bill but I couldn't addition of all the price.